tests/feature_test.py

```diff
@@ -3,34 +3,64 @@
 from utilities_common.db import Db
 
 show_feature_status_output="""\
-Feature     State     AutoRestart
-----------  --------  -------------
-bgp         enabled   enabled
-database    enabled   disabled
-dhcp_relay  enabled   enabled
-lldp        enabled   enabled
-nat         enabled   enabled
-pmon        enabled   enabled
-radv        enabled   enabled
-restapi     disabled  enabled
-sflow       disabled  enabled
-snmp        enabled   enabled
-swss        enabled   enabled
-syncd       enabled   enabled
-teamd       enabled   enabled
-telemetry   enabled   enabled
+Feature     State     AutoRestart    SystemState    UpdateTime    ContainerId    ContainerVersion    CurrentOwner    RemoteState
```

Comment:
what is the definition of SystemState. it is not defined in the design document.

Reply:
i think we can say Version instead of Container version here.

Reply:
I feel like user would like to get owner/currentowner at the same time for comparision.

can you put like

```
Owner CurrentOwner
------- -----------------
kube    local
```

Reply:
- Let me add "systemState" to doc -[here](https://github.com/Azure/SONiC/blob/be12cc665c316348352b868f515714f202861f63/doc/kubernetes/Kubernetes-support.md#state-db
)

- yes s/Container version/Version/

- This combines config & state in one o/p (just for FYI). 
-- @lguohan : Call it "set owner" instead of "owner" as just owner can be confusing to some ?

Reply:
done